PowerShell.sublime-syntax

```diff
@@ -793,3 +794,17 @@ contexts:
         3: support.variable.drive.powershell
         4: variable.other.readwrite.powershell
         5: punctuation.section.braces.end
+
+  class-method:
+    - match: ^(?:\s*)(?i)(hidden|static)?\s*(\[)((?!\d+|\.)[\p{L}\p{N}.]+)(\])\s*((?:\p{L}|\d|_|-|\.)+)\s*(?=\()
+      scope: meta.function.powershell
```

Comment:
Should the meta scope apply to the whole method definition, or just to the method name?